pkg/ccl/backupccl/restore_job.go

```diff
@@ -660,6 +660,27 @@ func shouldPreRestore(table *tabledesc.Mutable) bool {
 	return ok
 }
 
+// eligibleInProgressImport returns true if the descriptor represents a table with an in progress
```

Comment:
nit: can we maybe clarify that this method is used on "backed up" descriptors and not descriptors in the cluster. To that effect what do you think about renaming this to `backedUpDescriptorsWithInProgressImportInto`, a bit wordy but maybe its okay.

Reply:
Done.